docs/it/metadata_oidc_op_cie.rst

```diff
@@ -0,0 +1,264 @@
+.. include:: ./common_definitions.rst
+
+
+.. _claim_OP_CIE:
+
+
+OpenID Connect Provider Metadata (OP) per CIE
```

Comment:
@sp640919 rimuovere questo

gli OP SPID e CIE hanno gli stessi claim

in CIE tls_client_certificate_bound_access_tokens è deprecato
authorization_response_iss_parameter_supported mettilo come opzionale per adesso

@fmarino-ipzs ^